torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -2225,6 +2225,48 @@ RegisterOperators reg2({
 
     DEFINE_INT_OP(aten::gcd, gcd(a, b)),
 
+    DEFINE_GENERIC_OP(
+        aten::isnan,
+        std::isnan(a),
+        std::isnan(a),
+        float,
+        float),
+
+    DEFINE_GENERIC_OP(
+        aten::asinh,
+        std::asinh(a),
+        std::asinh(a),
+        float,
+        float),
+
+    DEFINE_GENERIC_OP(
+        aten::atanh,
```

Comment:
This should fail when the input is >=1 and needs to be tested accordingly

Reply:
I believe this is done and will push soon

Reply:
Complete